### DISCUSSION TITLE
topNavigation() widht

### DISCUSSION THREAD

**Discussion by dominikgeimer** [filamentphp/filament#8353] [Help]
Hi,

I really like the topNavigation(), but is it possible to change the width? Currently it is full width, but would be nice if it could be the same with as the main content.

Thanks Dominik

**Comment by leandrocfe** [comment]
You can [customize the css](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)

```css
.fi-topbar ul{
  @apply mx-auto md:px-6 lg:px-8 max-w-7xl;
}
```
